README.md

```diff
@@ -143,8 +143,7 @@ ___
 
 </div>
 
-[build-badge]: https://img.shields.io/travis/PyConColombia/website-2021.svg?style=flat-square
-[build]: https://travis-ci.org/PyConColombia/website-2021
+[build-badge]: https://github.com/PyConColombia/website-2021/workflows/Develop%20CI/badge.svg?branch=develop
```

Comment:
@arendondiosa  where/when is this badge built?

Reply:
@jonathanvm13 check this https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository

This badge is built by GitHub